hr_timesheet_holidays/wizard/holidays_import.py

```diff
@@ -60,18 +44,17 @@ def get_utc_end_of_day(date_str, local_tz):
     return get_utc_datetime(get_end_of_day(date_str), local_tz)
 
 
-class HolidaysImport(orm.TransientModel):
+class HolidaysImport(models.TransientModel):
```

Comment:
I understand the idea of a wizard but why not create the entry in the analytic account when the holiday is validated?
Would there be any reason it would not be possible?